apps/browser/browserservice.cpp

```diff
@@ -23,15 +23,19 @@
 
 #define SAILFISH_BROWSER_UI_SERVICE QLatin1String("org.sailfishos.browser.ui")
 
-#define IS_PRIVILEGED \
+#define GET_PID connection().interface()->servicePid(message().service())
```

Comment:
nit: macro that looks like constant, but does complex call sequence -> GET_PID() would feel more appropriate